OSMGeoweek-Nov-2017/README.md

```diff
@@ -0,0 +1,54 @@
+# OSMGeoweek Mapping party 
+*Mapbox, Bengaluru, 17 Nov 2017*
+
+#### Presented by:
+
+**Oormila Vinod** - oormila@mapbox.com
+
+**Chethan Gowda** - chethan@mapbox.com
+
+| | | 
+|---|---|
+|![gxb-500px-openstreetmap_logo](https://user-images.githubusercontent.com/6770741/28453477-34d856f8-6e15-11e7-95d1-022ab54b0355.png) | ![mapillary_logo svg](https://user-images.githubusercontent.com/6770741/32936973-6cc75bf6-cb9c-11e7-8701-83b2d155939c.png)|
+
+---
+
+- [OSMGeoweek.org](http://osmgeoweek.org/)
+- [Event page](https://www.mapbox.com/events/osm-geoweek-blr/)
+- [Facebook page](https://www.facebook.com/events/1239288609549971/)
+
+---
+
+## Getting started
+
+- Sign up [here](https://www.openstreetmap.org), if you are just getting started with OpenStreetMap.  
+- Navigate to [Jayanagar, Bengaluru](https://www.openstreetmap.org/#map=15/12.9390/77.5859) locality to get started with mapping using [Mapillary](https://www.mapillary.com/). 
+
+## Introduction to mapping using iD editor
+
+- Introduction to `node(point)`, `way(line)`, and `area(closed polygon)`.
+
+![pasted image at 2017_07_20 10_19 pm](https://user-images.githubusercontent.com/6770741/28456397-c7b56830-6e1f-11e7-9404-3838bfbcbd9e.png)
+- Here is the [video on how to use Mapillary in iD editor](https://www.youtube.com/watch?v=1H8fx6ZUDTw) to improve OpenStreetMap
+- How to map point's of interest(POI's) using Mapillary 
+
+![poimapping](https://imgur.com/a/Ii8vO)
+
+- Save your work and upload to OpenStreetMap.
+
+
+#### How to add Points of Interests (POI's) in OpenStreetMap?
+
+For example:
+- To add an hotel, use `amenity=restaurant` - https://wiki.openstreetmap.org/wiki/Tag:amenity%3Drestaurant
+- To add an ATM use `amenity=atm` - https://wiki.openstreetmap.org/wiki/Tag:amenity%3Datm
+- To add clinic use `amenity=clinic` - https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dclinic
+- To add toilets use `amenity=toilets` - https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dtoilets
+
+### Resources
+
+- [Map Features](https://wiki.openstreetmap.org/wiki/Map_Features#Amenity)
+- [LearnOSM](http://learnosm.org/en/) will help you get started with OpenStreetMap
+- [Mapping guides](https://www.mapbox.com/mapping/) 
+- [OSM mailing lists](https://wiki.openstreetmap.org/wiki/Mailing_lists) - subscribe to talk-in and participate in various discussions
+- [TagInfo](https://taginfo.openstreetmap.org/) - one place to find information about the tags used in OpenStreetMap
```

Comment:
Add Mapillary tutorial here as well. Anything else that you think they should read and will be helpful

Reply:
I'm thinking to add this http://blog.mapillary.com/tutorials/2015/01/13/getting_started.html Mapillary blog here? thoughts

Reply:
@chtnha 👍